crates/bevy_ecs/src/schedule/stage.rs

```diff
@@ -332,6 +345,45 @@ impl SystemStage {
             info!("{}", string);
         }
     }
+
+    /// Checks for old component and system counters
+    fn check_counters(&mut self, world: &mut World) {
```

Comment:
Given that this implementation hinges on wraparound working as expected. We should probably have unit tests for that.

Reply:
Added in 954086ccd2741a10ae4784c4e3c527bcf7c07a44.